lib/matplotlib/ticker.py

```diff
@@ -1353,13 +1356,30 @@ def format_pct(self, x, display_range):
             decimals = self.decimals
         s = '{x:0.{decimals}f}'.format(x=x, decimals=int(decimals))
 
-        if self.symbol:
-            return s + self.symbol
-        return s
+        return s + self.get_symbol()
 
     def convert_to_pct(self, x):
         return 100.0 * (x / self.xmax)
 
+    def get_symbol(self):
```

Comment:
How about making `self.symbol` into a property since it's already public and removing the need for this method entirely?